packages/client/src/components/MovieList/MovieList.jsx

```diff
@@ -0,0 +1,197 @@
+      movie_year: 2014,
+      image_location:
+        'https://www.themoviedb.org/t/p/w1280/qymaJhucquUwjpb8oiqynMeXnID.jpg',
+      price: 120,
+    },
+    {
+      id: 9,
+      category_id: 5,
+      title: 'The Exorcist',
+      description:
+        "Ophelia, a young nun recently arriving in the town of San Ramon, is forced to perform an exorcism on a pregnant woman in danger of dying. Just when she thinks her possession has ended, she discovers that the evil presence hasn't disappeared yet",
+      movie_year: 2014,
+      image_location:
+        'https://www.themoviedb.org/t/p/w1280/d07xtqwq1uriQ1hda6qeu8Skt5m.jpg',
+      price: 666,
+    },
+    {
+      id: 10,
+      category_id: 3,
+      title: 'The Dark Knight',
+      description:
+        'Batman raises the stakes in his war on crime. With the help of Lt. Jim Gordon and District Attorney Harvey Dent, Batman sets out to dismantle the remaining criminal organizations that plague the streets. The partnership proves to be effective.',
+      movie_year: 2008,
+      image_location:
+        'https://www.themoviedb.org/t/p/w1280/z4XwvnDZSm40PCdg9vFJmo0W9ri.jpg',
+      price: 120,
+    },
+  ];
+  return (
+    <Box bgcolor="mainGreen" fontFamily="Inter" marginTop="150rem">
+      <Typography
+        sx={{
+          position: 'relative',
+          height: '1.87rem',
+          width: '17.81rem',
+          top: '1.56rem',
+          left: '35rem',
+          fontFamily: 'Inter',
+          fontStyle: 'normal',
+          fontWeight: 'bold',
+          fontSize: '1.56rem',
+          color: '#003C2D',
+        }}
+        variant="h1"
+      />
+      <Grid container spacing={2}>
+        {dummyData.map((each) => (
+          <Grid key={each.id} item md={4} xs={12} sm={4}>
+            <CardMedia
+              sx={{
+                position: 'relative',
+                height: '27.12rem',
+                width: '18rem',
+                left: '10%',
+                top: '40%',
+                paddingTop: '8rem',
+                marginBottom: '4rem',
+                paddingLeft: '2.62rem',
+              }}
+              image={`${each.image_location}`}
+              title="Image Title"
+            >
+              <Button
+                variant="text"
+                sx={{
+                  position: 'absolute',
+                  height: '1.75rem',
+                  top: '0',
+                  right: '0',
+                }}
+              >
+                <FavoriteBorderIcon
+                  sx={{
+                    display: 'block',
+                    position: 'absolute',
+                    width: '1.9rem',
+                    height: '1.75rem',
+                    right: '0',
+                    color: '#FFFFFF',
+                    backgroundColor: '#000000CC',
+                    zIndex: 1,
+                  }}
+                />
+              </Button>
+              <Button
+                href={`/movies/:${each.id}`}
```

Comment:
the colon is obsolete here, output will be like this from your code (for example): `/movies/:9` and we want `/movies/9`

one small fix more! I will approve it now, but please make sure to fix this typo :)

Reply:
@magdazelena fixed it, thanks for notifying me.